app/assets/javascripts/modules/cross-domain-tracking.js

```diff
@@ -1,47 +1,55 @@
+window.GOVUK = window.GOVUK || {}
 window.GOVUK.Modules = window.GOVUK.Modules || {};
 
-(function (Modules, global) {
-  'use strict'
-
-  var $ = global.$
+(function (Modules) {
+  function CrossDomainTracking ($module) {
+    this.$module = $module
+    Modules.crossDomainLinkedTrackers = Modules.crossDomainLinkedTrackers || []
+  }
 
-  Modules.crossDomainLinkedTrackers = []
+  CrossDomainTracking.prototype.init = function () {
+    if (this.isTrackable(this.$module)) {
```

Comment:
non-blocker: alternatively, we could have used `bind` to preserve the instance (and be able to call `this.$module` from) within the `isTrackable` function (similarly for `addLinkedTrackerDomain`)

Reply:
Both `isTrackable` and `addLinkedTrackerDomain` are called later in the code and pass different elements. I considered trying something in order to not have to pass things, but decided that was more in the realm of a refactor than the scope of this PR.